fusor-ember-cli/app/templates/components/pf-modal-base.hbs

```diff
@@ -0,0 +1,15 @@
+<div class="modal fade" id="{{idModal}}" tabindex="-1" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+
+        <div class="modal-header">
+          <button type="button" class="close" {{action 'closeModal'}}>
```

Comment:
replaced bootstrap boilerplate `data-dismiss="modal"` with `{{action 'closeModal'}}` so it calls an action to toggle `openModal` to `false`